pkg/operator/ceohelpers/bootstrap.go

```diff
@@ -62,14 +64,16 @@ func GetBootstrapScalingStrategy(staticPodClient v1helpers.StaticPodOperatorClie
 		return strategy, fmt.Errorf("couldn't determine etcd unsupported override status, assuming default HA scaling strategy: %w", err)
 	}
 
+
 	etcdNamespace, err := namespaceLister.Get(operatorclient.TargetNamespace)
 	if err != nil {
 		return strategy, fmt.Errorf("failed to get %s namespace: %w", operatorclient.TargetNamespace, err)
 	}
 	_, hasDelayedHAAnnotation := etcdNamespace.Annotations[DelayedHABootstrapScalingStrategyAnnotation]
+	_, hasNonHAAnnotation := etcdNamespace.Annotations[NonHABootstrapScalingStrategyAnnotation]
```

Comment:
if we will have an api describing `None` now we should use it. assisted-install now will be the outlier to use annotation,